examples/browser/Dockerfile.dev

```diff
@@ -0,0 +1,22 @@
+#
+# Build image:
+#    docker build -t theia-dev -f Dockerfile.dev .
+#
+# How to use this image
+#
+# theia-dev image helps make and run theia without installing anything on the host
+# 1.  Build theia in docker:
+#     docker run -e GITHUB_TOKEN="<token>" -v /path/to/theia-ide-sources-folder/:/home/theia theia-dev sh -c 'cd /home/theia; yarn'
+#
+#     Note that sometimes you can reach Github API rate limit, so to solve this issue env var GITHUB_TOKEN could be used as per example
```

Comment:
Is using a GH token applicable to theia-dev? It seems this image does not fetch from GitHub and instead relies on having access to the source code from the host machine. Move this to the other image?

Reply:
this is needed only for `theia-dev` because during the build it making a lot of call to GH API, I added this because after few builds I got the API limit exceeded error. In the `theia` image it also could be useful as there is same build pase executed. So doc should be updated also.

Reply:
I also added mention about `GITHUB_TOKEN` in `theia` image

Reply:
cool, thanks.

Reply:
@marcdumais-work I've changed a bit `theia` dockerfile I was used `GITHUB_TOKEN` in wrong place on docker run while it is actually needed in that case only on docker build step. 
So to pass GH_TOKEN env I used docker ARG and updated docs how to use that. Please check this if that clear enough.